src/impl.cpp

```diff
@@ -448,6 +431,8 @@ void Manifold::Impl::CreateHalfedges(const Vec<ivec3>& triVerts) {
       if (h0.startVert != h1.endVert || h0.endVert != h1.startVert) break;
       if (halfedge_[NextHalfedge(pair0)].endVert ==
           halfedge_[NextHalfedge(pair1)].endVert) {
+        h0 = {-1, -1, -1};
+        h1 = {-1, -1, -1};
```

Comment:
I think this got lost accidentally during the `face` removal - it meant opposed triangles were not actually getting removed. I randomly noticed this on an SDF a couple weeks ago, so that should be fixed now too.